doc/filters/trim.rst

```diff
@@ -14,13 +14,26 @@ and end of a string:
 
     {# outputs '  I like Twig' #}
 
+    {{ '  I like Twig.  '|trim(type='left') }}
```

Comment:
You should add a `versionadded` tag at the beginning of the file (like done in `doc/filters/convert_encoding.rst` for instance):

```
.. versionadded:: 1.32
    The ``type`` argument was added in Twig 1.32.
```